docs/source/en/tasks/semantic_segmentation.md

```diff
@@ -14,15 +14,184 @@ rendered properly in your Markdown viewer.
 
 -->
 
-# Semantic segmentation
+# Image Segmentation
 
 [[open-in-colab]]
 
 <Youtube id="dKE8SIt9C-w"/>
 
-Semantic segmentation assigns a label or class to each individual pixel of an image. There are several types of segmentation, and in the case of semantic segmentation, no distinction is made between unique instances of the same object. Both objects are given the same label (for example, "car" instead of "car-1" and "car-2"). Common real-world applications of semantic segmentation include training self-driving cars to identify pedestrians and important traffic information, identifying cells and abnormalities in medical imagery, and monitoring environmental changes from satellite imagery.
+Image segmentation models divide images into segments consisting of different objects. These models segment images by assigning a label to each pixel. It has three different types: semantic segmentation, instance segmentation and panoptic segmentation.
```

Comment:
Similar to @pcuenca's note - it's not always objects that are being labelled depending on the type of segmentation happening.

```suggestion
Image segmentation models divide images into areas of interest such as different objects or background regions e.g. the sky. These models segment images by assigning a label to each pixel. There are three different types: semantic segmentation, instance segmentation, and panoptic segmentation.
```